--- a/nginx-module.control.in	2018-08-07 21:56:45.440486000 +0000
+++ b/nginx-module.control.in	2018-08-07 19:17:19.000000000 +0000
@@ -12,7 +12,7 @@
 Standards-Version: 3.9.8.0
 Homepage: %%MODULE_PACKAGE_URL%%
 
-Package: nginx-module-%%MODULE%%
+Package: %%MODULE_PACKAGE_NAME%%
 Architecture: any
 Depends: ${misc:Depends},
          ${shlibs:Depends},
@@ -20,12 +20,12 @@
 Description: %%SUMMARY%%
  %%SHORT_SUMMARY%% for nginx
 
-Package: nginx-module-%%MODULE%%-dbg
+Package: %%MODULE_PACKAGE_NAME%%-dbg
 Architecture: any
 Section: debug
 Priority: extra
 Depends: nginx (= %%MAIN_VERSION%%-%%MAIN_RELEASE%%~%%CODENAME%%),
-         nginx-module-%%MODULE%% (= %%VERSION_PREFIX%%%%MODULE_VERSION%%-%%MODULE_RELEASE%%~%%CODENAME%%),
+         %%MODULE_PACKAGE_NAME%% (= %%VERSION_PREFIX%%%%MODULE_VERSION%%-%%MODULE_RELEASE%%~%%CODENAME%%),
          ${misc:Depends}
 Description: debug symbols for the nginx-module-%%MODULE%%
  %%SHORT_SUMMARY%% for nginx (debug symbols)
