--- a/Makefile	2018-08-07 21:56:45.436483999 +0000
+++ b/Makefile	2018-08-07 19:07:52.000000000 +0000
@@ -347,6 +347,7 @@
 	cat $(MODULE_CONTROL_TEMPLATE) | sed \
 		-e "s#%%MODULE%%#$${module_asis}#g" \
 		-e "s#%%MODULE_PACKAGE_VENDOR%%#$(MODULE_PACKAGE_VENDOR)#g" \
+		-e "s#%%MODULE_PACKAGE_NAME%%#$(MODULE_PACKAGE_NAME_$(call modname, $@))#g" \
 		-e "s#%%MODULE_PACKAGE_URL%%#$(MODULE_PACKAGE_URL)#g" \
 		-e "s#%%SUMMARY%%#$(MODULE_SUMMARY_PREFIX) $(MODULE_SUMMARY_$(call modname, $@))#g" \
 		-e "s#%%SHORT_SUMMARY%%#$(MODULE_SUMMARY_$(call modname, $@))#g" \
@@ -370,6 +371,7 @@
 	> $${builddir}/$(SRCDIR)/debian/changelog ; \
 	cat nginx-module.rules.in | sed \
 		-e "s#%%MODULE%%#$${module_asis}#g" \
+		-e "s#%%MODULE_PACKAGE_NAME%%#$(MODULE_PACKAGE_NAME_$(call modname, $@))#g" \
 		-e "s#%%CODENAME%%#$(CODENAME)#g" \
 		-e "s#%%MAIN_VERSION%%#$(BASE_VERSION)#g" \
 		-e "s#%%MAIN_RELEASE%%#$(BASE_RELEASE)#g" \
@@ -382,7 +384,7 @@
 	> $${builddir}/$(SRCDIR)/debian/rules ; \
 	cat nginx-module.postinst.in | sed \
 		-e "s#%%MODULE_POST%%#$$post#g" \
-	> $${builddir}/$(SRCDIR)/debian/nginx$(TARGET_SUFFIX)-module-$${module_asis}.postinst ; \
+	> $${builddir}/$(SRCDIR)/debian/$(MODULE_PACKAGE_NAME_$(call modname, $@)).postinst ; \
 	chmod +x $${builddir}/$(SRCDIR)/debian/rules ; \
 	ln -s $${builddir}/$(SRCDIR)/debian/rules $@ ; \
 	}
